composer.json

```diff
@@ -74,40 +75,43 @@
         }
     },
     "scripts": {
+        "bootstrap-test-environment": [
+            "Tests/Application/bin/adminconsole doctrine:database:create --if-not-exists --env test",
+            "Tests/Application/bin/adminconsole doctrine:schema:update --force --env test"
+        ],
         "lint": [
-            "@lint-composer",
-            "@lint-yaml",
-            "@lint-twig",
-            "@lint-schema",
             "@phpstan",
-            "@lint-container"
-        ],
-        "lint-composer": "@composer validate --strict",
-        "lint-yaml": "@php Tests/Application/bin/adminconsole lint:yaml Resources/config",
```

Comment:
removed this because the `Resources/config` folder does not contain any yaml files

Reply:
okay for me